src/App.jsx

```diff
@@ -14,57 +16,58 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedValue, SetValue] = useState(goods[8]);
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods[8]);
```
1. the variable should describe what exactly it stores (be self-descriptive)
2. camelCase for variables, functions. In JS+React we use PascalCase for classes, constructor-functions and React components
3. state values should always be named like `[someName, setSomeName]`